reactive_graph/src/traits.rs

```diff
@@ -61,6 +61,7 @@ use std::{
     panic::Location,
 };
 
+#[doc(hidden)]
 /// Provides a sensible panic message for accessing disposed signals.
 #[macro_export]
 macro_rules! unwrap_signal {
```

Comment:
If you want to make a declarative macro internal within the crate (which seems to be intended), you can use:
```rs
// No macro_export
macro_rules! unwrap_signal {...}
pub(crate) use unwrap_signal;
```

Reply:
Thanks, TIL! Searching though I find that I do actually use it in `reactive_stores` for exactly the same purpose... so I will probably just hide it for now.